Packs/QSS/pack_metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "SOC Case Management Platform",
```

Comment:
Please change the name to Quantum Security Systems

Reply:
Done